index.html

```diff
@@ -48,63 +58,81 @@ <h1>Tonic</h1>
                 </ul>
                 <button class="see_project">See project</button>
             </div>
+            </div>
+            <!-- card two -->
             <div class="card2 card">
-                <img src="/imgs/Snapshoot Portfolio(3).png" alt="">
+                <div class="card-img">
+                    <img src="/imgs/Snapshoot Portfolio(3).png" alt="">
+                </div>
+                <div class="card-info">
                 <h1>Multi-Post
                     Stories</h1>
                 <ul class="first_list">
                     <li>CANOPY</li>
-                    <li>Back End Dev</li>
-                    <li>2015</li>
+                    <li class="dim_color_text">&#9679; Back End Dev</li>
+                    <li class="dim_color_text">&#9679; 2015</li>
                 </ul>
                 <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                 <ul class="second_list">
                     <button>html</button>
                     <button>css</button>
+                    <button>Ruby on rails</button>
                     <button>javascript</button>
                 </ul>
                 <button class="see_project">See project</button>
+                </div>
             </div>
+            <!-- card three -->
             <div class="card3 card">
-                <img src="/imgs/Snapshoot Portfolio.png" alt="">
-                <h1>Tonic</h1>
+                <div class="card-img">
+                    <img src="/imgs/Snapshoot Portfolio22.png" alt="">
+                </div>
+                <div class="card-info">
+                <h1>Facebook 360</h1>
                 <ul class="first_list">
                     <li>CANOPY</li>
-                    <li>Back End Dev</li>
-                    <li>2015</li>
+                    <li class="dim_color_text">&#9679; Back End Dev</li>
+                    <li class="dim_color_text">&#9679; 2015</li>
                 </ul>
                 <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                 <ul class="second_list">
                     <button>html</button>
                     <button>css</button>
+                    <button>Ruby on rails</button>
                     <button>javascript</button>
                 </ul>
                 <button class="see_project">See project</button>
```

Comment:
**OPTIONAL** - Good job done so far, but you seem to have styled this button wrongly, please check the Figma file and apply the right styling for this button and all project buttons, please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project:
![46he46y](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/73607512/ad3f4fd8-1a6e-4080-9d33-8265bf4e5b25)

Figma template:
![hzerhzer](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/73607512/9f7de08e-044f-4c76-b6b7-d2fcf62efb8e)